frontend/src/routes/subRoutes/projectRoutes.tsx

```diff
@@ -5,63 +5,17 @@ import { PageLink } from "../../components/PageLink";
 import { Project, Projects } from "../../pages";
 
 const projectRoutes = [
-  <Route key="project" path="/projects" element={<ProtectedRoute component={Projects} />}>
-    <Route path=":projectId" element={<ProtectedRoute component={Project} />} />
-    <Route
-      path=":projectId/status"
-      element={
-        <ProtectedRoute
-          component={() => {
-            return PageLink("Status");
-          }}
-        />
-      }
-    />
-    <Route
-      path=":projectId/change-request"
-      element={
-        <ProtectedRoute
-          component={() => {
-            return PageLink("ChangeRequest");
-          }}
-        />
-      }
-    />
-    <Route
-      path=":projectId/billing"
-      element={
-        <ProtectedRoute
-          component={() => {
-            return PageLink("Billing");
-          }}
-        />
-      }
-    />
-    ,
-    <Route
-      key="projectLessonsLearned"
-      path=":projectId/lessons-learned"
-      element={
-        <ProtectedRoute
-          component={() => {
-            return PageLink("LessonsLearned");
-          }}
-        />
-      }
-    />
-    ,
-    <Route
-      key="projectCloseOut"
-      path=":projectId/close-out"
-      element={
-        <ProtectedRoute
-          component={() => {
-            return PageLink("CloseOut");
-          }}
-        />
-      }
-    />
-  </Route>,
+  <Route key="project" path="projects" element={<Projects />} />,
```

Comment:
so i guess the reason for route changes is because of protected route... again, this should be indicated to the reviewer ie DESCW-381, this should have been its own pr.

Reply:
Things were done a bit out of scope of the ticket... The things that were completed outside of the ticket were needed for the core functionality of the button like routing.  Next time a new ticket should be made to address these out of scope items.